polyfill.js

```diff
@@ -0,0 +1,137 @@
+'use strict'
+
+if (typeof Symbol !== 'function' || !Symbol.isConcatSpreadable) {
+  throw new Error('Symbol.isConcatSpreadable missing from the environment')
+}
+
+if (typeof Array.prototype.flatMap !== 'function') {
+  // eslint-disable-next-line no-extend-native
```

Comment:
These eslint comments are due to, when I originally wrote the polyfill, I was working on a repository with `eslint-config-standard`. Should I also add it here, remove eslint comments, or what?